docs/rum_collection/README.md

```diff
@@ -0,0 +1,131 @@
+---
+title: RUM iOS Monitoring
+kind: documentation
+beta: true
+description: "Collect RUM data from your iOS applications."
+further_reading:
+  - link: "https://github.com/DataDog/dd-sdk-ios"
+    tag: "Github"
+    text: "dd-sdk-ios Source code"
+  - link: "/real_user_monitoring"
+    tag: "Homepage"
+    text: "Explore Datadog RUM"
+---
+
+Datadog *Real User Monitoring (RUM)* enables you to visualize and analyze the real-time performance and user journeys of your application's individual users.
+
+## Setup
+
+1. Declare SDK as a dependency.
+2. Specify application details in UI.
+3. Initialize the library.
+4. Initialize RUM Monitor, `DDURLSesssionDelegate` and start sending data.
+
+**Minimum iOS version**: Datadog SDK for iOS supports iOS v11+.
+
+### Declare SDK as dependency
+
+1. Declare [dd-sdk-ios][1] as a dependency, depending on your package manager.
+
+
+| Package manager            | Installation method                                                                         |
+|----------------------------|---------------------------------------------------------------------------------------------|
+| [CocoaPods][2]             | `pod 'DatadogSDK'`                                                                          |
+| [Swift Package Manager][3] | `.package(url: "https://github.com/DataDog/dd-sdk-ios.git", .upToNextMajor(from: "1.0.0"))` |
+| [Carthage][4]              | `github "DataDog/dd-sdk-ios"`                                                               |
+
+### Specify application details in UI
+
+1. Select UX Monitoring -> RUM Applications -> New Application
+2. Choose `iOS` as your Application Type in [Datadog UI][2] and provide a new application name to generate a unique Datadog application ID and client token.
+
+{{< img src="real_user_monitoring/ios/screenshot_rum.png" alt="RUM Event hierarchy" style="width:50%;border:none" >}}
+
+To ensure safety of your data, you must use a client token: you cannot use [Datadog API keys][6] to configure the `dd-sdk-ios` library as they would be exposed client-side in the iOS application byte code. For more information about setting up a client token, see the [client token documentation][7].
```

Comment:
```suggestion
To ensure safety of your data, you must use a client token, **not** [Datadog API keys][6] to configure the `dd-sdk-ios` library because they would be exposed client-side in the iOS application byte code. For more information about setting up a client token, see the [client token documentation][7].
```